### ISSUE TITLE
Fixate Timeline

### ISSUE THREAD

**Issue by albertotimoteo** [almende/vis#2552]
How can I fixate my div timeline, so it'll be always on the bottom of the screen?

**Comment by yotamberk** [comment]
Add to your css (assuming the dive has a class of `timeline`)
 ``` css
.timeline {
 position: fixed;
 Bottom: 0;
}
```

**Comment by Tooa** [comment]
Does this answer your question @albertotimoteo? Then, feel free to close this issue.

**Comment by albertotimoteo** [comment]
It doesn't work, I have tried this before, and then I tried again. Vis simply doesn't draw my timeline and gives me a warning: 
`Something is wrong with the Timeline scale. Limited drawing of grid lines to 1000 lines.
TimeAxis._repaintLabels @ vis.js:25198
TimeAxis.redraw @ vis.js:25084
(anonymous) @ vis.js:19556
Core._redraw @ vis.js:19555
(anonymous) @ vis.js:779`

I created a div just like the page examples: 
`<div id="timeline" class="timeline"></div>`

And added the style tag at the head of my page:
`<style>
        .timeline {
            position: fixed;
            bottom: 0;
        }
   </style>`

Without the style the timeline works fine, but with it, the console gives that warning and doesn't draw anything.
